build/cloudbuild_tools.yaml

```diff
@@ -13,7 +13,7 @@
 #  limitations under the License.
 
 substitutions:
-  _KUBERNETES_VERSION: 1.26.1
```

Comment:
TO update the k8s versions we need to:
- update controller-runtime and controller-tools (then do new releases)
- generate an new kubebuilder-tools for it (envtest)
- update the scaffolds 
- bump the deps above

Reply:
More info: https://github.com/kubernetes-sigs/kubebuilder/issues/3054


/hold

Reply:
c/c @rashmigottipati ^